wpimath/BUILD.bazel

```diff
@@ -99,10 +99,17 @@ filegroup(
     srcs = glob(["src/generated/main/java/**/*.java"]),
 )
 
-third_party_cc_lib_helper(
-    name = "eigen",
-    include_root = "src/main/native/thirdparty/eigen/include",
+cc_library(
+    name = "eigen-headers",
     visibility = ["//visibility:public"],
+    deps = [
+        "@eigen",
+    ],
+)
```

Comment:
Is this target necessary? Doesn't really seem to do anything.

Reply:
The sugar rules PJ added want something called name + "-headers".  This adapts for them.

Reply:
Why can't this be an `alias()` then?

Reply:
OK, should be good now.